### PR TITLE
Update max LP fee to 20%

### PR DESCRIPTION
**Pull request by C-o-d-e-C-o-w-b-o-y**
# Summary

Increase max LP fee from `10%` -> `20%`

Also fix bug where wrong error code was being used for LP fee issue (`InvalidLPFee` was not used anywhere in program)

# Test Plan

Screenshot 2024-03-28 at 9 24 41 AM

### CODE REVIEW

**Review by nothing0012 — APPROVED** [review]

File: programs/mmm/src/errors.rs [hunk 1]
```diff
@@ -2,7 +2,7 @@ use anchor_lang::prelude::*;
 
 #[error_code]
 pub enum MMMErrorCode {
-    #[msg("lp fee bp must be between 0 and 10000")]
+    #[msg("lp fee bp must be between 0 and 20000")]
```
Comment (nothing0012) [inline programs/mmm/src/errors.rs:5]: nit: 2000

File: sdk/src/idl/mmm.ts [hunk 2]
```diff
@@ -2411,7 +2411,7 @@ export type Mmm = {
     {
       "code": 6000,
       "name": "InvalidLPFee",
-      "msg": "lp fee bp must be between 0 and 10000"
+      "msg": "lp fee bp must be between 0 and 20000"
```
Comment (nothing0012) [inline sdk/src/idl/mmm.ts:2414]: nit: 2000

File: programs/mmm/src/errors.rs (hunk 1 above)
Comment (C-o-d-e-C-o-w-b-o-y) [reply programs/mmm/src/errors.rs]: Amended 🙏

File: sdk/src/idl/mmm.ts (hunk 2 above)
Comment (C-o-d-e-C-o-w-b-o-y) [reply sdk/src/idl/mmm.ts]: Amended 🙏

**Review by nothing0012 — APPROVED** [review]

**Pull request merged**